lib/Alembic/CMakeLists.txt

```diff
@@ -61,7 +61,9 @@ TARGET_INCLUDE_DIRECTORIES(Alembic
     )
 
 IF (ALEMBIC_SHARED_LIBS)
-  SET_TARGET_PROPERTIES(Alembic PROPERTIES DEFINE_SYMBOL ALEMBIC_EXPORTS)
+  SET_TARGET_PROPERTIES(Alembic PROPERTIES DEFINE_SYMBOL ALEMBIC_EXPORTS
+  SOVERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}
+  VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH})
```

Comment:
you can use directly `${PROJECT_VERSION}` as declared here https://github.com/alembic/alembic/blob/master/CMakeLists.txt#L55